ebusd/DOCS.md

```
@@ -65,12 +67,12 @@ and you are inside the docker container, where you can easily run `ebusctl` or `
 
 You can add any command line options using the custom command line options field.  Check the eBUSd wiki for all available options - https://github.com/john30/ebusd/wiki/2.-Run
 
-For example ``` --initsend --dumpconfig```
+For example ` --initsend --dumpconfig`
```

Markdownlint (0.37.0)

Spaces inside code span elements

(MD038, no-space-in-code)